plugins/extraction/extract_env.ml

```diff
@@ -644,8 +644,12 @@ let separate_extraction lr =
 (*s Simple extraction in the Coq toplevel. The vernacular command
     is \verb!Extraction! [qualid]. *)
 
+let dump_qualid r =
+  let gr = Smartlocate.global_with_alias r in
+  Dumpglob.add_glob gr
+
 let simple_extraction r =
-  Vernacentries.dump_global CAst.(make (Constrexpr.AN r));
+  dump_qualid r;
```

Comment:
IMO we could just remove the dump, the sibling functions (full_extraction etc) don't bother for instance.
Or we could dump after locate_ref so we don't have to duplicate the locate call for dumping.
Or even dump *inside* locate_ref? oO What will science come up with next

Reply:
Umm, that's for sure a naîve question, but why does the locate fail?

Reply:
It looks for globrefs but we're calling extraction on a module.

Reply:
Ok, let's remove the dump then.